resources/docs/analyses/relauncher.md

```diff
@@ -0,0 +1,14 @@
+This service automatically relaunches analyses with the IDs given in the request.
+
+* If a requested analysis was a sub-job of an HT analysis,
+  then the relaunched analysis will also be a sub-job of that parent HT analysis.
+* Any of the original analyses that have the `create_output_subdir` flag set to `false` will cause the corresponding
```

Comment:
This might be obvious, but if `create_output_subdir` is set to `true`, then what happens?

Reply:
It will behave the same as submitting a new job to the [POST /analyses](https://qa.cyverse.org/terrain/docs/index.html#!/analyses/post_terrain_analyses) endpoint.
I guess those docs could be more clear, but if the optional `create_output_subdir` flag is omitted in the `POST /analyses` request, then it defaults to `true`, which means the service will auto-create an output folder under the path in the request (`/iplant/home/user/analyses` for example). So in that case, data will not be overwritten, and this endpoint doesn't have to do anything different from the `POST /analyses` endpoint.

Reply:
Cool, thanks for explaining this - makes sense!